include/net/net_context.h

```diff
@@ -878,6 +878,19 @@ int net_context_get_option(struct net_context *context,
 			   enum net_context_option option,
 			   void *value, size_t *len);
 
+/**
+ * @brief Find matching network context with given input values.
+ *
+ * @param family IP address family (AF_INET or AF_INET6 or AF_PACKET)
+ * @param type Type of the socket, SOCK_STREAM or SOCK_DGRAM or SOCK_RAW
```

Comment:
Too many "or" here, just say `@param type Type of the network socket. Values is either SOCK_STREAM, SOCK_DGRAM or SOCK_RAW`

Reply:
ok